sycl/source/detail/event_impl.cpp

```diff
@@ -303,6 +312,9 @@ template <> cl_uint event_impl::get_info<info::event::reference_count>() const {
 template <>
 info::event_command_status
 event_impl::get_info<info::event::command_execution_status>() const {
+  if (MState == HES_Invalid)
```

Comment:
Maybe it's better to say `discarded` rather than `invalid`? Invalid sounds like something wrong with the implementation.

Reply:
Ok, I will change `HES_Invalid` to `HES_Discarded`